components/match_ticker/commons/match_ticker.lua

```diff
@@ -136,15 +136,18 @@ function MatchTicker:init(args)
 
 	local wrapperClasses = type(args.wrapperClasses) == 'table' and args.wrapperClasses
 		or args.wrapperClasses == NONE and {}
-		or {args.wrapperClasses or WRAPPER_DEFAULT_CLASS}
+		or {args.wrapperClasses or args.wrapperClass}
```

Comment:
`args.wrapperClass` is not a valid input according to annotations, doesn't seem to be used anywhere else neither?

Reply:
yes
can not really annotate args easily due to having several where it is not known how many are needed (tournamentX etc)

Reply:
But it's not used anywhere else, is it correct? Should it not be `args.infoboxWrapperClass`?